collector/asp_net_application.go

```diff
@@ -0,0 +1,160 @@
+package collector
+
+import (
+	"github.com/prometheus-community/windows_exporter/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/yusufpapurcu/wmi"
+)
+
+func init() {
+	registerCollector("asp_net_application", newasp_net_applicationCollector) // TODO: Add any perflib dependencies here
+}
```

Comment:
You'll want to remove any `init()` functions in the collectors in favour of the `collectors` variable in `collector/init.go`.

If there's no perflib dependencies you can remove the TODO comment.